ProjectK.Data/ExperienceRepository.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using ProjectK.Business.ExperiencePoints;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectK.Data
+{
+    public class ExperienceRepository : IExperienceRepository
+    {
+        private readonly ProjectKContext _context;
+
+        public ExperienceRepository(ProjectKContext context)
+        {
+            _context = context;
+        }
+
```

Comment:
Empty space.

Reply:
This is a 1:1 copy from CharacterRepository (line 12-16). Is that also "Empty Space"?

Reply:
https://deviq.com/principles/boy-scout-rule